Header header
float32[] Qdot_meas
float32[] FOF_OF_SF
float32 FOF_threshold
float32 FOF_yaw_rate_cmd
float32[] FR_Qdot_SF
float32 FR_yaw_rate_cmd
float32 FR_threshold 
